website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md

```diff
@@ -262,3 +323,9 @@ Users in your IdP will now be able to log into the application by navigating to
 ### Setting up RBAC
 After configuring an identity provider, you will be able to set up [role-based
 access control](/access-control/enterprise-permissions) for your account.
+
+**Please note the following for Google integrations -**
+* While it is possible to integrate with Google Workspace via SAML, because Google's current SAML implementation does not support expressing group membership information, we strongly recommend the use of our native Google Workspace integration.
```

Comment:
Check the formatting here -- I think we want to NOT have lines 328-329 with bullets, but have lines 330 and 331 with bullets.